internal/codeintel/codenav/utils.go

```diff
@@ -117,3 +117,13 @@ func sortRanges(ranges []types.Range) []types.Range {
 
 	return ranges
 }
+
+func dedupeRanges(ranges []types.Range) []types.Range {
+	dedup := ranges[:1]
```

Comment:
```suggestion
	if len(ranges) == 0 {
		return ranges
	}

	dedup := ranges[:1]
```

If `ranges` is empty this will panic.